flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/TaskExecutorTest.java

```diff
@@ -1662,6 +1665,82 @@ public void testTerminationOnFatalError() throws Throwable {
 		}
 	}
 
+	/**
+	 * Tests that task manager will sync slot with job master throw heartbeat (See FLINK-11059).
```

Comment:
`Tests that the TaskExecutor syncs its slots view with the JobMaster's view via the SlotSnapshot reported by the heartbeat (See FLINK-11059).`

Reply:
OK